src/lib/fetch.ts

```diff
@@ -9,6 +9,8 @@ import APIError from './apiError';
 import isGHA from './isGitHub';
 import { debug } from './logger';
 
+const SUCCESS_NO_CONTENT = 204;
```

Comment:
The `DELETE /api/v1/docs/{slug}` endpoint returns 204 with an empty body if the deletion was successful.